main.ts

```diff
@@ -404,6 +414,37 @@ class Translator {
         this.emit('}');
         break;
 
+      case ts.SyntaxKind.ImportDeclaration:
+        var importDecl = <ts.ImportDeclaration>node;
+        this.emit('import');
+
+        this.visitExternalModuleReferenceExpr(importDecl.moduleSpecifier);
+
+        if (importDecl.importClause) {
+          this.emit('show');
+          this.visit(importDecl.importClause);
+        }
```

Comment:
What's the semantic of "import 'foo';" in ES6? In Dart, this imports all symbols into the global scope, which I think doesn't match what happens in Dart (where it barfs all over your global scope). I think we should outlaw that and just fail for import without an import clause.